app/Http/Controllers/CostumerController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Costumer;
```

Comment:
tipo dikit kyk waktu bikin model wkwk 
```suggestion
use App\Models\Customer;
```